src/pyproject_fmt/formatter/pep508.py

```diff
@@ -39,18 +39,25 @@ def _best_effort_string_repr(req: str) -> String:
         return toml_string(req)
 
 
-def normalize_pep508_array(requires_array: Array | None, indent: int) -> None:
+def normalize_pep508_array(requires_array: Array | None, indent: int, *, preserve_dependency_versions: bool) -> None:
```

Comment:
The `*` is here to avoid https://docs.astral.sh/ruff/rules/boolean-type-hint-positional-argument/.